PodcastGenerator/core/backwards.php

```diff
@@ -69,6 +69,8 @@ function backwards_3_1_to_3_2($absoluteurl)
 
 \$podcast_description = \"".$config['podcast_description']."\";
 
+\$podcast_cover = \"".$config['podcast_cover']."\";
```

Comment:
Check to see if `$config['podcast_cover']` has a value, and write out itunes_image.jpg if it doesn't:

```php
\$podcast_cover = \"".(isset($config['podcast_cover']) ? $config['podcast_cover'] : 'itunes_image.jpg')."\";
```